src/components/common/UserForm.js

```diff
@@ -9,31 +9,26 @@ function UserForm({
   setFormVisibility,
   formVisibility,
 }) {
-  const [userValue, setUserValue] = useState();
-
-  const handleSubmit = value => {
-    console.log('submitted!!!', value);
-    // set component state
-    setUserValue(value);
-
+  const handleSubmit = e => {
+    console.log('submit event!!!');
```

Comment:
Are these `console.log()s` still needed for the future functionality or can they be removed at this stage?